pkg/reconciler/pingsource/pingsource.go

```diff
@@ -21,6 +21,8 @@ import (
 	"encoding/json"
 	"fmt"
 
+	"knative.dev/pkg/tracing/config"
```

Comment:
Please group with other Knative `pkg` packages (line 37)

Reply:
thanks! changing it now